addons/l10n_mx/__manifest__.py

```diff
@@ -33,6 +33,7 @@
     "depends": [
         "account",
     ],
+    'post_init_hook': 'add_debit_tag',
```

Comment:
Is the post_init_hook called when you update your module?

Reply:
ping @ZedarkMoon :slightly_smiling_face: